src/CrosschainDeployScript.sol

```diff
@@ -59,31 +64,47 @@ contract CrosschainDeployScript is Script {
         _stringToNetworkIds["mumbai"] = NetworkIds(7, 80001, Env.TESTNET);
         _stringToNetworkIds["arbitrum-sepolia"] = NetworkIds(8, 421614, Env.TESTNET);
         _stringToNetworkIds["gnosis-chiado"] = NetworkIds(9, 10200, Env.TESTNET);
+
+        _domainIdToDeploymentTargets[2] = "sepolia";
+        _domainIdToDeploymentTargets[5] = "cronos-testnet";
+        _domainIdToDeploymentTargets[6] = "holesky";
+        _domainIdToDeploymentTargets[7] = "mumbai";
+        _domainIdToDeploymentTargets[8] = "arbitrum-sepolia";
+        _domainIdToDeploymentTargets[9] = "gnosis-chiado";
```

Comment:
maybe it would be better to have domainId -> chainId mapping and some internal method
`addNetwork` which would set both _stringToNetworkIds and domainId -> chainId mappings. You could also remove chainId from _stringToNetworkIds

Reply:
I leave that choice on you :)